src/style.css

```
@@ -0,0 +1,23 @@
+html,
+body {
+  margin: 0;
+}
```

why do you need `html` in this selector? 
```suggestion
body {
  margin: 0;
}
```